### PR TITLE
Adding compact option to Table layout

### PR DESCRIPTION
**Pull request by mvehar-preskok**
## Proposed Changes

Adding compact Table layout (compact bootstrap table)

### CODE REVIEW

**Comment by tabuna** [comment]
Have you checked it? For me, adding the class didn't change anything. Probably CSS was not overridden

**Comment by mvehar-preskok** [comment]
There was an issue with ccs rules priority. Decided to add new class that copies the original table-sm + adding important. 
I could also use exact css rule as the one that originally prevented the table-sm to work.

P.s. Should I include all built files in PR? vendor.js, orchid.js, orchid.css, etc.?

**Comment by tabuna** [comment]
Please don't include js/css build files in your PR

File: resources/sass/core/tables.scss [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@import '~bootstrap/scss/variables';
```
Comment (tabuna) [inline resources/sass/core/tables.scss:1]: Variables are already loaded earlier. You don't have to re-upload them.

**Pull request merged**